docker-api/README.md

```diff
@@ -14,7 +14,7 @@ But if you really need a minimum benchmark, we have seen the Sirius Blockchain P
 
 This README was prepared by testing the package on:
 - Debian 10 ++
-- Ubuntu 19.04 ++
+- Ubuntu 18.04 ++
```

Comment:
why downgrade to 18?

Reply:
i thought with the `++` means it should work with 18.04 onwards.  the impression i get is:
18.04++ = compatible with 18.04, 18.10, 19.04, 19,10, 20.04
19.04++ = compatible with 19.04, ,19.10, 20.04

also, 19.04 is not LTS